projects/js-packages/number-formatters/eslint.config.mjs

```diff
@@ -0,0 +1,27 @@
+// @ts-ignore
+import makeBaseConfig from 'jetpack-js-tools/eslintrc/base.mjs';
+
+export default [
+	...makeBaseConfig( import.meta.url ),
+	{
+		languageOptions: {
+			parserOptions: {
+				project: './tsconfig.json',
+			},
+		},
+		files: [ '**/*.ts', '**/*.tsx' ],
+		plugins: {
+			'@typescript-eslint': true,
+			jsdoc: true,
+		},
+		rules: {
+			'jsdoc/require-jsdoc': 'off',
+			'jsdoc/require-description': 'off',
+			'jsdoc/require-returns': 'off',
+			'jsdoc/require-returns-check': 'off',
+			'jsdoc/require-param': 'off',
+			'jsdoc/require-param-description': 'off',
+			'jsdoc/require-param-type': 'off',
```

Comment:
Personally I'd recommend you not disable all these, as good documentation will help reusers know how to use things and these set the low bar of "documentation exists" for that. But I won't try to force it if you really don't want these.

P.S. Our base config already turns off `jsdoc/require-param-type` for TypeScript files (as well as `jsdoc/require-property-type` and `jsdoc/require-returns-type`), since TypeScript already provides type declarations.

Reply:
I think this was just the result of battling it out to remove `jsdoc/require-returns`. We don't require params and return definitions in the JS docs in Calypso, as those are rather accurately defined from TS types for the most part. In cases where ambiguity might exist e.g. a string return that's the transformation of something, I'd normally define these, but in general, I'd just rely on TS.

Happy to remove these if it's a convention in JP.

Reply:
For TypeScript code, our standard configuration shouldn't require the type be specified in the jsdoc but will still want a human-readable description of the parameters and return value. See files in `projects/js-packages/publicize-components/` for example.

Reply:
Fair. Removed (upcoming changes)